Automattic/vip-go-mu-plugins
# Use es-wp-query for offloading

## Description

Autoloads es-wp-query (if not already loaded elsewhere) and the VIP Search adapter, for query offloading when the `'es'` query var is set.

Disables sending `'es'` queries to ElasticPress.

Depends on https://github.com/Automattic/vip-go-mu-plugins/pull/1594

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

1. Check out PR.
1. Offload a query, like this:
```
add_filter( 'pre_get_posts', function( $query ) {
	if ( ! $query->is_main_query() ) { 
		return $query;
	}

	if ( $query->is_category() || $query->is_home() ) {
		$query->set( 'es', true );
	}
} );
```

1. On the HP or a category page, you should see the same posts, but an ES query should be registered in the debug bar, and the main query replaced with one that just grabs posts by id.
